core/modules/hubotModule.js

```diff
@@ -0,0 +1,133 @@
+﻿/**
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Irregular whitespace not allowed](https://www.codacy.com/app/theprojectshop/Kassy/file/2530854170/issues/source?bid=3282987&fileBranchId=3283570#l1)